test/Sentry.Tests/Internals/Http/HttpTransportTests.cs

```diff
@@ -10,6 +10,8 @@ namespace Sentry.Tests.Internals.Http;
 
 public class HttpTransportTests
 {
+    private static readonly IDiagnosticLogger DebugLogger = new TraceDiagnosticLogger(SentryLevel.Debug);
```

Comment:
The school of thought that drove the author or these tests is that tests should be as independent of external state as possible. Trade off is there's more duplication but often moving things around in this "test classes state" doesn't cause other tests to break.

Obviously harder to vary behavior by "centrally" changing setup. You'd need to find and replace or manually change each occurrence. Since in this pr no test changed the instance, neither a different type of diag logger nor a different level, I personally don't see a reason to make this change. That said, tests I write indeed have central state I mutate before each test so I'm most certainly not against the approach used here.

Reply:
is the instance of IDiagnosticLogger is not important for the tests. should we have a static no-op instance of IDiagnosticLogger  ?

Reply:
`TraceDiagnosticLogger` is stateless other than the minimum level, which is the constant passed in the constructor, and it's immutable.  It just forwards everything to `Trace.Writeline`, so there's really no point in multiple instances floating around.  It was just a quick cleanup.

That said, I'm not sure why we're using it here.  xUnit stopped supporting `Trace` and `Debug` in v2.x. We should actually be using `ITestOutputHelper`.  See https://xunit.net/docs/capturing-output.  I'll make another adjustment for that.